docs/_articles/en/lwc/writing.md

```diff
@@ -29,76 +29,82 @@ To learn more about `@salesforce` modules, see [`@salesforce` Modules](https://d
+![Documentation on Hover](../../../images/vscode_lwc_hover.png)
 
 ## Linting
 
 Linting provides errors about malformed or suspicious-looking code while you edit. VS Code enforces Salesforce's ESLint rules. To activate ESLint, install it from the command line. See the [Lightning Web Components ESLint Plugin](https://github.com/salesforce/eslint-plugin-lwc) and [Lightning Web Components ESLint Config](https://github.com/salesforce/eslint-config-lwc) repositories for instructions.
 
 In this example, when hovering over `onpress`, the linter reports that you cannot name an API property starting with "on."
 
-![Linter Example with Hover](./images/vscode_lwc_linting_press.png)
+![Linter Example with Hover](../../../images/vscode_lwc_linting_press.png)
 
 ### Peek Problem and Quick Fix
 
 VS Code has a range of actions to quickly address problems and refactor code, including Quick Fix and Peek Problem. To learn more, see [Refactoring](https://code.visualstudio.com/docs/editor/refactoring) in the VS Code documentation.
 
 Clicking **Peek Problem** on the error message highlights the line where the linter found the error. If the message says there is more than one error, click the down arrow in the message's upper right corner to see the others.
 
-![Linter Example with Peek](./images/vscode_lwc_peek.png)
+![Linter Example with Peek](../../../images/vscode_lwc_peek.png)
 
 Clicking **Quick Fix** provides options to disable the warning on valid API names in the line or in the file, and links to the documentation. You can see these same options by clicking on the yellow lightbulb icon next to `@api onpress;`.
 
-![Linter Example with Quick Fix](./images/vscode_lwc_quickfix.png)
+![Linter Example with Quick Fix](../../../images/vscode_lwc_quickfix.png)
 
 ### Errors and Warnings
 
 File and compiler errors display on hover. Here, when `@track` is hovered over, the error message says that it must be declared.
 
-![Documentation on Hover](./images/vscode_lwc_track.png)
+![Documentation on Hover](../../../images/vscode_lwc_track.png)
 
 ## Code Navigation
 
 VS Code provides shortcuts to preview or jump to definitions within your code without losing track of the code you're currently working on. Learn more in the [Code Navigation](https://code.visualstudio.com/docs/editor/editingevolved) section of the VS Code documentation.
 
 To preview a definition, hold down **Ctrl** (Windows or Linux) or **Command** (macOS) and hover over the item whose definition you want to see. This example shows a preview of the source of a `c` namespace component.
 
-![View source of a c namespace component](./images/vscode_lwc_commandhover.png)
+![View source of a c namespace component](../../../images/vscode_lwc_commandhover.png)
 
 To view a definition, right-click the item and select **Peek Definition**, or press **Alt+F12**.
 
 To jump to the location of a definition, right-click the item and select **Go to Definition**, or press **F12**.
+
+## Supporting js-meta.xml
+
```

Comment:
##Support for js-meta.xml

VS Code provides IntelliSense support by integrating the Red Hat plug-in for `js-meta.xml`. It supports features such as auto suggestion, syntax error reporting, renaming support, and automatic code generation.

@lcampos - do you have any feedback/edits? Is this from XML Language Server extension? if yes, we could substitute "plug-in" above for the actual name.

Reply:
@ekapner yes, this is a feature that depends on the user installing the [Red Hat XML](https://marketplace.visualstudio.com/items?itemName=redhat.vscode-xml) extension. I think we should replace `plug-in` with `extension` or `VSCode extension`.